### PR TITLE
[linstor] fixed DRBD kernel module load

### PR DESCRIPTION
**Pull request by duckhawk**
## Description

Changed the method of loading DRBD kernel module on the nodes with active LINSTOR satellites.

## Why do we need it, and what problem does it solve?

On some recent versions of CentOS-based distributions, the old method of loading DRBD has stopped working.

## Why do we need it in the patch release (if we do)?

Some of our clients occasionally experience issues due to DRBD's module not loaded in CentOS-based distros. 

## What is the expected result?

Correctly loaded DRBD module on any OS version.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: linstor
type: fix
summary: Changed the method of loading DRBD kernel module on the nodes with active LINSTOR satellites.
```

### CODE REVIEW

File: modules/041-linstor/templates/nodegroupconfiguration-drbd-install-altlinux-like.yaml [hunk 1]
```diff
@@ -83,6 +83,11 @@ spec:
       fi
 
       if [ "${current_version}" == "${desired_version}" ]; then
+        if [ $(egrep -e "^drbd$" "/etc/modules" | wc -l) -ne 0 ]; then
```
Comment (apolovov) [inline modules/041-linstor/templates/nodegroupconfiguration-drbd-install-altlinux-like.yaml:86]:
```suggestion
        if grep -E '^drbd$' /etc/modules; then
```

File: modules/041-linstor/templates/nodegroupconfiguration-drbd-install-centos-like.yaml [hunk 2]
```diff
@@ -88,6 +88,11 @@ spec:
       fi
 
       if [ "${current_version}" == "${desired_version}" ]; then
+        if [ $(egrep -e "^drbd$" "/etc/modules" | wc -l) -ne 0 ]; then
```
Comment (apolovov) [inline modules/041-linstor/templates/nodegroupconfiguration-drbd-install-centos-like.yaml:91]:
```suggestion
        if grep -q -E '^drbd$' /etc/modules; then
```

File: modules/041-linstor/templates/nodegroupconfiguration-drbd-install-centos-like.yaml [hunk 3]
```diff
@@ -134,7 +139,10 @@ spec:
     make install
     cd drbd
     cp *.ko /lib/modules/${kernel_version_in_use}/kernel/drivers/
-    echo 'drbd' | tee -a /etc/modules
+    if [ $(egrep -e "^drbd$" "/etc/modules" | wc -l) -ne 0 ]; then
```
Comment (apolovov) [inline modules/041-linstor/templates/nodegroupconfiguration-drbd-install-centos-like.yaml:142]:
```suggestion
    if grep -q -E '^drbd$' /etc/modules; then
```

File: modules/041-linstor/templates/nodegroupconfiguration-drbd-install-debian-like.yaml [hunk 4]
```diff
@@ -85,6 +85,11 @@ spec:
       fi
 
       if [ "${current_version}" == "${desired_version}" ]; then
+        if [ $(egrep -e "^drbd$" "/etc/modules" | wc -l) -ne 0 ]; then
```
Comment (apolovov) [inline modules/041-linstor/templates/nodegroupconfiguration-drbd-install-debian-like.yaml:88]:
```suggestion
        if grep -q -E '^drbd$' /etc/modules; then
```

**Review by apolovov — APPROVED** [review]

**Review by AleksZimin — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [6287](https://github.com/deckhouse/deckhouse/pull/6287) to the branch [release-1.53](https://github.com/deckhouse/deckhouse/tree/release-1.53) successful!

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [6288](https://github.com/deckhouse/deckhouse/pull/6288) to the branch [release-1.52](https://github.com/deckhouse/deckhouse/tree/release-1.52) successful!

**Pull request merged**